seatunnel-connectors/seatunnel-connector-flink-kafka/src/main/java/org/apache/seatunnel/flink/sink/KafkaTable.java

```diff
@@ -38,6 +38,8 @@
 import org.apache.flink.table.descriptors.Schema;
 import org.apache.flink.types.Row;
 
+import javax.annotation.Nullable;
+
 import java.util.Properties;
```

Comment:
Does the checkstyle rules for `javax` work?

Reply:
Yes, it does